scripts/getconfig.go

```diff
@@ -658,7 +659,7 @@ func main() {
 	outOfDateFlags := checkFlag(allFlags, flagsToCheck)
 	updatedArgs := updateArgsClean(args, outOfDateFlags)
 	updatedArgs = updateArgsAdd(updatedArgs)
-	updatedArgs = handleGRPC(args, updatedArgs)
+	// updatedArgs = handleGRPC(args, updatedArgs)
```

Comment:
why we'd like to comment out gRPC?

Reply:
Well this is the migration script which when provided with an old start.sh spits out a new cli based toml config file. Earlier we used to set `grpc.addr = pprof.addr + ":" + pprof.port" as the grpc address. But now as that port is taken by default prometheus config, we would have to rely on a different port. I suggest relying on the default port (3131) instead of anything else. 

Also, it would be difficult to change prometheus port as all of our nodes have service running on that port.